main.js

```diff
@@ -0,0 +1,38 @@
+const key = "4b5774e9f3d2a07b84f0f2f88e486224";
+const url = new URL("https://api.openweathermap.org/data/2.5/weather");
+document.addEventListener('DOMContentLoaded', function () {
+    let place = document.getElementById('place').value;
+    call(place);
+    document.getElementById('place').addEventListener('change', function () {
+        place = document.getElementById('place').value;
+        call(place);
+    });
+});
+
+async function call(place) {
```

Comment:
async functionもアロー関数にしましょう
アロー関数を定義したら末尾の ; もお忘れなく
```suggestion
const call = async (place) => {
```